.github/actions/next-integration-stat/src/index.ts

```diff
@@ -233,45 +161,9 @@ async function getJobResults(
     }
   )
 
-  // Filter out next.js build setup jobs
-  const nextjsBuildSetupJob = jobs?.find((job) =>
-    /Build Next.js for the turbopack integration test$/.test(job.name)
-  )
-
-  // Next.js build setup jobs includes the version of next.js that is being tested, try to read it.
-  const nextjsVersion = await findNextJsVersionFromBuildLogs(
-    octokit,
-    token,
-    nextjsBuildSetupJob
-  )
-
-  // Find out next-swc build workflow
-  const nextSwcBuildJob = jobs?.find((job) =>
-    job.name.includes('Build Next.js for the turbopack integration test')
-  )
-  const nextSwcBuildLogs = (
-    await fetchJobLogsFromWorkflow(octokit, token, nextSwcBuildJob)
-  ).logs.split('\n')
-  const buildTimeMatch = (
-    nextSwcBuildLogs.find((line) => line.includes('Time (abs ≡):')) ?? ''
-  ).match(/  ([+-]?(?=\.\d|\d)(?:\d+)?(?:\.?\d*))(?:[Ee]([+-]?\d+))? s/)
-  const buildTime = buildTimeMatch?.length >= 2 ? buildTimeMatch[1] : undefined
-  const nextSwcBuildSize = (
```

Comment:
This seems like it would be nice to have binary size reporting, but it should live in the build-and-test workflows, and not here.

Plus, we're not currently using the comment-on-PR functionality for this workflow anyways, so it's pointless to try to support this code.